src/models/baichuan.h

```diff
@@ -40,3 +40,19 @@ class Baichuan : public CommonDecoder<BaichuanAttention<WeiT, LlamaRotaryEmbeddi
     TokenEmbedding<float16_t> *embedding;
     RmsNorm finalLN;
 };
+
+REGISTER_DECODER(Baichuan, baichuan, float)
```

Comment:
Move to cpp file is better, suppose 2 cpp files include baichuan.h, then the model will be registered twice.

Reply:
Yes, it will be registered twice in the cpp example. But I don't know how to trigger these static object initializations if they are in the cpp files..